<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarshipMainWindow</class>
 <widget class="QMainWindow" name="StarshipMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Starship</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="HexTileGraphicsView" name="hexView">
      <property name="dragMode">
       <enum>QGraphicsView::RubberBandDrag</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>&amp;Simulation</string>
    </property>
    <addaction name="actionSimRunning"/>
    <addaction name="actionSimulationParameters"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionStatusBar"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionBuildDeck"/>
    <addaction name="actionBuildBulkhead"/>
    <addaction name="separator"/>
    <addaction name="actionUnbuild"/>
    <addaction name="separator"/>
    <addaction name="actionCancelJob"/>
    <addaction name="separator"/>
    <addaction name="actionQuickDeck"/>
    <addaction name="actionQuickBulkhead"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBuild"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="simulationParameters">
   <property name="windowTitle">
    <string>Simulation Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="DockContentsWidget" name="simulationDockContents">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>289</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="simRateLabel">
        <property name="text">
         <string>Limit simulation rate to:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="simRateBaseLabel">
          <property name="text">
           <string>60^</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="simRateLimit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>-99.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99.989999999999995</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="simRateLimitUnitsLabel">
          <property name="text">
           <string>seconds per second</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="frameRateLabel">
        <property name="text">
         <string>Limit frame rate to:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="frameRateLimit">
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="singleStep">
           <number>15</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="frameRateUnitsLabel">
          <property name="text">
           <string>frames per second</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Exit this application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Starship</string>
   </property>
   <property name="statusTip">
    <string>Display Starship version and copyright</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="statusTip">
    <string>Display Qt version and copyright</string>
   </property>
  </action>
  <action name="actionSimRunning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Running</string>
   </property>
   <property name="statusTip">
    <string>Pause or resume the flow of time</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Status Bar</string>
   </property>
   <property name="statusTip">
    <string>Toggle visibility of this status bar</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionSimulationParameters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Parameters panel</string>
   </property>
  </action>
  <action name="actionBuildDeck">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Deck</string>
   </property>
  </action>
  <action name="actionBuildBulkhead">
   <property name="text">
    <string>&amp;Bulkhead</string>
   </property>
  </action>
  <action name="actionUnbuild">
   <property name="text">
    <string>&amp;Un-build</string>
   </property>
  </action>
  <action name="actionCancelJob">
   <property name="text">
    <string>&amp;Cancel Job</string>
   </property>
  </action>
  <action name="actionQuickDeck">
   <property name="text">
    <string>Quick Deck</string>
   </property>
  </action>
  <action name="actionQuickBulkhead">
   <property name="text">
    <string>Quick Bulkhead</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexTileGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>hextilegraphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>DockContentsWidget</class>
   <extends>QWidget</extends>
   <header>dockcontentswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>StarshipMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStatusBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>statusbar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSimulationParameters</sender>
   <signal>toggled(bool)</signal>
   <receiver>simulationParameters</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simulationParameters</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionSimulationParameters</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
